src/sentry/api/endpoints/release_thresholds/release_threshold_index.py

```diff
@@ -31,6 +32,9 @@ class ReleaseThresholdIndexEndpoint(OrganizationEndpoint):
     }
 
     def get(self, request: Request, organization: Organization) -> HttpResponse:
+        if not features.has("organizations:release-ui-v2", organization, actor=request.user):
```

Comment:
I'm not worried about this case since none of these apis are active at all 🤷 
If you feel strongly i can remove for clarity, but order of deploy here won't make any difference. this was just an after thought i tossed in 😅

Reply:
actually

I'll just remove this altogether.
it's not really necessary